From 33c314f0d97b2e35914d0c8958138595112ea1c2 Mon Sep 17 00:00:00 2001
From: WanWizard <wanwizard@openpli.org>
Date: Wed, 7 May 2025 18:27:58 +0000
Subject: [PATCH] updated to python 3.12

---
 bt_config.py |  2 +-
 gbbt.py      | 26 +-------------------------
 2 files changed, 2 insertions(+), 26 deletions(-)

diff --git a/bt_config.py b/bt_config.py
index 89768d1..aec4418 100644
--- a/bt_config.py
+++ b/bt_config.py
@@ -4,7 +4,7 @@ from Components.config import config, ConfigSelection, getConfigListEntry, Confi
 from Components.ConfigList import ConfigListScreen
 from Components.Sources.StaticText import StaticText
 
-class BluetoothSetupConfig(Screen, ConfigListScreen):
+class BluetoothSetupConfig(ConfigListScreen, Screen):
 	skin =	"""
 		<screen position="100,100" size="530,250">
 			<ePixmap pixmap="skin_default/buttons/red.png" position="45,10" size="140,40" alphatest="on" />
diff --git a/gbbt.py b/gbbt.py
index fbf3d5a..beff34a 100644
--- a/gbbt.py
+++ b/gbbt.py
@@ -4,32 +4,8 @@
 # Do not make changes to this file unless you know what you are doing--modify
 # the SWIG interface file instead.
 
+from . import _gbbt
 
-
-
-
-from sys import version_info
-if version_info >= (2, 6, 0):
-    def swig_import_helper():
-        from os.path import dirname
-        import imp
-        fp = None
-        try:
-            fp, pathname, description = imp.find_module('_gbbt', [dirname(__file__)])
-        except ImportError:
-            from . import _gbbt
-            return _gbbt
-        if fp is not None:
-            try:
-                _mod = imp.load_module('_gbbt', fp, pathname, description)
-            finally:
-                fp.close()
-            return _mod
-    _gbbt = swig_import_helper()
-    del swig_import_helper
-else:
-    from . import _gbbt
-del version_info
 try:
     _swig_property = property
 except NameError:
-- 
2.43.5

